### PR TITLE
[HW-2] Implement my array methods

### PR DESCRIPTION
**Pull request by den-kak-den**

### CODE REVIEW

File: HW02/Denis_Havanski/hw02.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+module HDArrayMethods
```
Comment (B0RlS) [inline HW02/Denis_Havanski/hw02.rb:1]: could you change the name of the module, it's not entirely clear.
Make the file name the same as the module name (snake case)

File: HW02/Denis_Havanski/hw02.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      end
+    end
+
+    def my_select
+      if block_given?
+        ret_array = []
+        for i in 0...self.size
+          if yield(self[i]) == true
+            ret_array << self[i]
+          end
```
Comment (B0RlS) [inline HW02/Denis_Havanski/hw02.rb:20]: use ternary conditional operation please

File: HW02/Denis_Havanski/hw02.rb (hunk 1 above)
Comment (den-kak-den) [reply HW02/Denis_Havanski/hw02.rb]: Done

File: HW02/Denis_Havanski/hw02.rb (hunk 2 above)
Comment (den-kak-den) [reply HW02/Denis_Havanski/hw02.rb]: Done

**Pull request merged**